modules/apps/web-experience/site-navigation/site-navigation-menu-web/src/main/resources/com/liferay/site/navigation/menu/web/portlet/template/dependencies/portlet_display_template_navbar_blank.ftl

```diff
@@ -28,7 +28,11 @@
 
 					<#if showChildren>
 						<#assign nav_item_attr_has_popup = "aria-haspopup='true'" />
-						<#assign nav_item_caret = '<span class="lfr-nav-child-toggle"><i class="icon-caret-down"></i></span>' />
+						<#assign nav_item_caret>
+							<span class="lfr-nav-child-toggle">
+								<@liferay_ui["icon"] icon="angle-down" markupView="lexicon"/>
```

Comment:
hey @natecavanaugh you are right, we don't need markupView="lexicon".
And yes, I tested it and it was working (new screenshot added), Eduardo helped me with this and he said me that I should use <@liferay_ui["icon"], would you rather change it?

![image](https://cloud.githubusercontent.com/assets/7963804/13990651/6380f646-f115-11e5-9591-74c4d09bd137.png)